nixos/modules/services/desktops/pipewire/wireplumber.nix

```diff
@@ -0,0 +1,43 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
```

Comment:
Might be cleaner to use explicit `lib.` prefix, see https://nix.dev/anti-patterns/language#with-attrset-expression.

Reply:
Fixed.